setup.py

```diff
@@ -92,13 +94,17 @@ def read(*rnames):
     keywords=keywords,
     packages=[
         "zope",
-        "zope.testing"],
+        "zope.testing",
+    ],
     package_dir={'': 'src'},
     namespace_packages=['zope'],
     install_requires=[
         'setuptools',
-        'zope.exceptions',
-        'zope.interface'],
+    ],
+    extras_require={
+        'test': [
```

Comment:
I was wondering, what is the benefit of an empty [test] requirement? It seems like boilerplate here, or is there something else coming?